src/EPL-2.0.xml

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+	<license isOsiApproved="true" licenseId="EPL-2.0"
+		name="Eclipse Public License 2.0">
+		<crossRefs>
+			<crossRef>http://www.eclipse.org/legal/epl-2.0</crossRef>
+			<crossRef>http://www.opensource.org/licenses/EPL-2.0</crossRef>
+		</crossRefs>
+		<notes></notes>
```

Comment:
If we don't have any notes on this license, I think we should leave these off ([they're optional][1]).

[1]: https://github.com/spdx/license-list-XML/blob/500c834cddce63632bb839fb5502575551753042/schema/ListedLicense.xsd#L22